script/setup/hcsshim-version

```diff
@@ -0,0 +1 @@
+v0.10.0
```

Comment:
I would call this `runhcs-version` instead, as that's the specific component we are using (hcsshim is the repo).

Reply:
Ack

Reply:
Done!